### PR TITLE
use package managers on mac and linux

### PR DESCRIPTION
**Pull request by jksolbakken**
merge when we have confirmed that https://github.com/nais/device/pull/189 works

### CODE REVIEW

File: docs/device/install.md [hunk 1]
```diff
@@ -6,11 +6,17 @@
...
 2. [Install the Kolide agent](#install-kolide-agent).
-3. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-4. Turn on your freshly installed `naisdevice` app.
+3. Install [Homebrew](https://brew.sh/) unless yoy already have it.
+4. Write in the terminal: 
+```bash
+brew tap nais/tap
+brew install naisdevice
+``` 
+(you'll might be prompted for your laptop-account's password to unlock `sudo`).
```
Comment (toby1knby) [inline docs/device/install.md:15]: The resulting action is the installation of a .pkg no? In that case the pkg should/will prompt for credentials and so "you'll might" should be replaced with "you will".

File: docs/device/install.md [hunk 2]
```diff
@@ -6,11 +6,20 @@
...
-3. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-4. Turn on your freshly installed `naisdevice` app.
+3. Install [Homebrew](https://brew.sh/) unless yoy already have it.
+4. Add the nais tap
+```bash
+brew tap nais/tap
+```
+5. Install the naisdevice cask: 
+```bash
+brew install naisdevice
```
Comment (mortenlj) [inline docs/device/install.md:16]: Er det ikke `brew cask install naisdevice`?

File: docs/device/install.md [hunk 3]
```diff
@@ -26,11 +35,19 @@
...
-2. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-3. Turn on your freshly installed `naisdevice` application.
+2. Add the nais PPA repo:
+``` 
+curl -s --compressed "https://ppa.nais.io/KEY.gpg" | sudo apt-key add -
+sudo curl -s --compressed -o /etc/apt/sources.list.d/my_list_file.list "https://ppa.nais.io/my_list_file.list"
+```
+3. Intall the naisdevice package:
+```
+apt install naisdevice
```
Comment (mortenlj) [inline docs/device/install.md:45]: Lurer på om du trenger en `apt update` etter at du har lagt til repoet..? Litt usikker.. :thinking:

Comment (jksolbakken) [reply docs/device/install.md:45]: Slenger på en, skader jo uansett ikke.

File: docs/device/install.md (hunk 2 above)
Comment (jksolbakken) [reply docs/device/install.md:16]: Ikke nå lenger, de droppa `cask` i en relativt ny versjon

File: docs/device/install.md [hunk 4]
```diff
@@ -26,11 +35,20 @@
     Using Gnome DE on latest Ubuntu LTS - only supported variant atm
 
 1. [Install Kolide agent](install.md#install-kolide-agent).
-2. Write in the terminal: `curl https://device.nais.io/install.sh | bash` \(you might be prompted for your laptop-account's password to unlock `sudo`\).
-3. Turn on your freshly installed `naisdevice` application.
+2. Add the nais PPA repo:
+``` 
+curl -s --compressed "https://ppa.nais.io/KEY.gpg" | sudo apt-key add -
+sudo curl -s --compressed -o /etc/apt/sources.list.d/my_list_file.list "https://ppa.nais.io/my_list_file.list"
```
Comment (x10an14) [inline docs/device/install.md:41]: Denne fungerer ikke for meg: https://ppa.nais.io/my_list_file.list <-- 404

Kunne jeg uansett foreslå:
```suggestion
curl -s --compressed -o "https://ppa.nais.io/my_list_file.list" | sudo tee /etc/apt/sources.list.d/my_list_file.list
```

**Review by x10an14 — CHANGES_REQUESTED** [review]
Funker ikke for meg =/

File: docs/device/install.md [hunk 5]
```diff
@@ -37,8 +37,10 @@ brew install naisdevice
 1. [Install Kolide agent](install.md#install-kolide-agent).
 2. Add the nais PPA repo:
 ``` 
-curl -s --compressed "https://ppa.nais.io/KEY.gpg" | sudo apt-key add -
-curl -s --compressed -o "https://ppa.nais.io/nav_nais.list" | sudo tee /etc/apt/sources.list.d/nav_nais.list
+NAIS_GPG_KEY="/usr/local/share/keyrings/nav_nais.gpg"
```
Comment (mortenlj) [inline docs/device/install.md:40]: Jeg trodde konvensjonen var å legge slike keys her: `/etc/apt/trusted.gpg.d/` ?

Comment (x10an14) [reply docs/device/install.md:40]: Fulgte denne https://unix.stackexchange.com/questions/332672/how-to-add-a-third-party-repo-and-key-in-debian/582853#582853

Har ingen sterk formening utover å hensynta dette: https://askubuntu.com/a/1307181/817936

**Review by x10an14 — APPROVED** [review]
Linux LGTM! =)

**Pull request merged**